### ISSUE TITLE
Missing translation example

### ISSUE THREAD

**Issue by Seb33300** [symfony/symfony-docs#10399]
On this page: https://symfony.com/doc/current/translation/debug.html

Just after

> If you delete the message Symfony is great from your translation file for the fr locale and run the command, you will get:

The result is a `missing` translation line because the related key was deleted from the `fr` locale file.
But if the translation was deleted from the `fr` locale file, how can the french translation `J'aime Symfony` could still be displayed in the result?
If it was deleted, Symfony can't know it.

Same for the next example.

Or maybe I am not understanding something :/

**Comment by xabbuh** [comment]
Good catch @Seb33300. That's indeed wrong. It should be "Symfony is great" instead as the following sentences also indicate:

> Moreover, the message in the fr locale equals to the message in the en locale. This is a special case because the untranslated message id equals its translation in the en locale.

Would you like to submit a PR to fix the documentation?

**Comment by Seb33300** [comment]
Maybe examples will be more clear if we use translation keys instead of english text?

**Comment by xabbuh** [comment]
That could make sense, but I think we should discuss that separately.

**Comment by Seb33300** [comment]
I just created the pull request to fix examples.

But it was hard for me to understand the difference between 2 last examples.
I had to test by myself with translation keys to understand.
